src/bootstrap/src/lib.rs

```diff
@@ -1777,9 +1779,8 @@ Executed at: {executed_at}"#,
             }
         }
         if let Ok(()) = fs::hard_link(&src, dst) {
-            // Attempt to "easy copy" by creating a hard link
-            // (symlinks don't work on windows), but if that fails
-            // just fall back to a slow `copy` operation.
+            // Attempt to "easy copy" by creating a hard link (symlinks are priviledged on windows),
+            // but if that fails just fall back to a slow `copy` operation.
```

Comment:
This was a drive-by comment update because symlinks are priviledge on Windows, but AFAIK they do work when you have the priviledge (but can be weird).